### PR TITLE
Setup spec-prod

### PR DESCRIPTION
**Pull request by saschanaz**
This requires some prior steps before merging:

* Renaming the current branch to main (rather than creating main, so that the existing PRs can be automatically redirected)
* Create gh-pages again
* Make sure the new gh-pages branch does not require approval for commits

***
<a href="https://pr-preview.s3.amazonaws.com/saschanaz/media-source/pull/270.html" title="Last updated on Apr 3, 2021, 9:41 PM UTC (dccafa2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/media-source/270/2a413f9...saschanaz:dccafa2.html" title="Last updated on Apr 3, 2021, 9:41 PM UTC (dccafa2)">Diff</a>

### CODE REVIEW

File: media-source-respec.html [hunk 1]
```diff
@@ -142,7 +142,9 @@
     <section id="introduction" class="informative">
       <h2>Introduction</h2>
       <p>This specification allows JavaScript to dynamically construct media streams for &lt;audio&gt; and &lt;video&gt;. It defines a MediaSource object that can serve as a source of media data for an HTMLMediaElement. MediaSource objects have one or more <a>SourceBuffer</a> objects.  Applications append data segments to the <a>SourceBuffer</a> objects, and can adapt the quality of appended data based on system performance and other factors. Data from the <a>SourceBuffer</a> objects is managed as track buffers for audio, video and text data that is decoded and played. Byte stream specifications used with these extensions are available in the byte stream format registry [[MSE-REGISTRY]].</p>
-      <img src="pipeline_model.svg" alt="Media Source Pipeline Model Diagram" longdesc='pipeline_model_description.html#pipelinedesc'>
+      <a href='pipeline_model_description.html#pipelinedesc'>
+        <picture><img src="pipeline_model.svg" alt="Media Source Pipeline Model Diagram"></picture>
```
Comment (saschanaz) [inline media-source-respec.html:146]: `<a href>` replaces `longdesc` which is currently no-op on every browser. [`<picture>` workarounds ReSpec width=150 issue](https://github.com/w3c/respec/issues/3435).

File: pipeline_model.svg [hunk 2]
```diff
@@ -4,7 +4,7 @@
     xmlns="http://www.w3.org/2000/svg"
     xmlns:svg="http://www.w3.org/2000/svg"
     xmlns:xlink="http://www.w3.org/1999/xlink"
-    viewBox="0 0 1500 1500">
+    viewBox="0 0 1400 1100">
```
Comment (saschanaz) [inline pipeline_model.svg:7]: This reduce needless empty space from the image.

**Comment by saschanaz** [comment]
@tidoust could you review this?

**Review by tidoust — APPROVED** [review]
That looks good to me.

@wolenetz, this removes the need to re-generate `index.html` manually each time an update gets made to `media-source-respec.html`.

The change of name for the default branch is mostly transparent, except that [branch names of local clones will have to be updated](https://docs.github.com/en/github/administering-a-repository/renaming-a-branch#updating-a-local-clone-after-a-branch-name-changes).

**Comment by tidoust** [comment]
Default branch name for the repo has been switched to `main`. I re-created `gh-pages` (and told GitHub to publish that branch). I don't think specific approvals are needed but I'll adjust if something breaks...

I'll go ahead and merge. Thanks @saschanaz!

**Comment by wolenetz** [comment]
Thank you for doing this, @saschanaz  and @tidoust

**Pull request merged**